centreon/www/class/centreon-clapi/centreonHost.class.php

```diff
@@ -764,6 +764,10 @@ public function initUpdateParameters($parameters = null)
                     $params[1] = 'host_location';
                     $params[2] = $iIdTimezone;
                     break;
+                case 'snmp_community':
+                    $params[1] = 'host_snmp_community';
+                    $params[3] === '1' ?: $params[3] = '0';
```

Comment:
For lisibility, I would have written only one assign, up to you to decide : matter of taste.
I am not very fond of the elvis operator "if shortcut", I prefer the ternary
```suggestion
                    $params[3] = $params[3] === '1' ? '1' : '0';
```